src/Config/packages-template.aget

```diff
@@ -13,8 +13,7 @@
       "Newtonsoft.Json": "8.0.3",
       "NUnit": "2.6.3",
       "Prism": "4.1.0",
-      "RestSharp": "106.12.0",
-      "Unofficial.Ionic.Zip": "1.9.1.8"
```

Comment:
@Mikhinja Please confirm it is fine to remove such dependency as part of DynamoRevit 2.19 release

Reply:
@QilongTang I checked the DynamoRevit addin and do not see this binary there.

Reply:
Also note that this PR is to the master branch - which I assume will not be subject to such reasons for 2 reasons.
1. potential 3.0
2. GL25 is supposed to resolve patch installer insanity.

Reply:
Doesn't look like it's actually used in DynamoRevit nodes, not sure why it's there.

Reply:
hmm @mjkkirschner I thought this change is for R2024.2?